exec/cnex/number.c

```diff
@@ -510,8 +512,31 @@ BOOL number_is_odd(Number x)
 
 //#define __NUMBER_TESTS
```

Comment:
These tests appear to be disabled in the build, and no build file appears to define `__NUMBER_TESTS`. Make sure these tests are enabled and run in the build.